octavia_f5/controller/worker/l2_sync_manager.py

```diff
@@ -48,6 +48,9 @@ def __init__(self):
         self.executor = futures.ThreadPoolExecutor(max_workers=CONF.networking.max_workers)
 
     def initialize_bigips(self, bigip_urls: [str]):
+        if CONF.f5_agent.dry_run:
+            return []
```

Comment:
Hrm, I don't like this, dry-run is not meant to completely ignore any (l2) action on devices, but rather not to do any modification but emitting correct logs.

It's true that dry-run hasn't been adapted for l2_sync_manager, but ignoring devices could be achieved by letting the  bigip_urls and vcmp_urls empty in the first place.